api/src/pcapi/core/educational/api/stock.py

```diff
@@ -82,7 +83,21 @@ def edit_collective_stock(
     if booking_limit is None:
         check_booking_limit_datetime = stock.bookingLimitDatetime
 
-    offer_validation.check_booking_limit_datetime(stock, check_beginning, check_booking_limit_datetime)
+    booking_status = [booking.status for booking in stock.collectiveBookings]
+    if not stock_data.get("beginningDatetime") and not stock_data.get("bookingLimitDatetime"):
```

Comment:
```suggestion
    if not "beginningDatetime" not in stock_data and "bookingLimitDatetime" not in stock_data:
```
sinon on s'expose a des bugs si ces champs contienne tdes valeurs equivalente a False